erlang/compile_script

```diff
@@ -0,0 +1 @@
+ erlc -o beam/ `find . -not -path './p2p/*' -name '*.erl'`
```

Comment:
:o

Reply:
-not -path './p2p/*' cause your stuff doesn't compile ಠ_ಠ

Reply:
:o

Reply:
you're the one who code reviewed it :o

Reply:
O: